ED/protocol.html

```diff
@@ -701,6 +701,8 @@ <h4 property="schema:name"><span>Note</span>: Self-describing Resources</h4>
                     <li><a href="#auxiliary-resources-description-resource">Resource Description</a></li>
                   </ul>
 
+                  <p><span about="" id="server-link-auxiliary-type" rel="spec:requirement" resource="#server-link-auxiliary-type"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> advertise auxiliary resources associated with a subject resource by responding to <code>HEAD</code> and <code>GET</code> requests including the HTTP <code>Link</code> header with the <code>rel</code> parameter [<cite><a class="bibref" href="#bib-rfc8288">RFC8288</a></cite>].</span></span></p>
```

Comment:
But what `rel` value?

Reply:
The `rel` value is given in the table below, but may also be defined in the future specifications implementing this behaviour. Is that sufficient?